app/controllers/output_elements_controller.rb

```diff
@@ -19,7 +19,7 @@ def batch
     ids = params[:ids].to_s.split(',').reject(&:blank?).uniq
 
     json = OutputElementPresenter.collection(
-      OutputElement.where(id: ids),
+      OutputElement.where(key: ids),
```

Comment:
I think this is a bit confusing. The variable is called ids in the params and as variable but used to hold keys.

Reply:
Agreed. This happens in more (actually a lot of) places, also in the chart js code, that's why I was a bit reluctant to change all ids into keys. It might take me a few hours to sort it all out. But I totally agree that it can be confusing to read `id` everywhere when we are talking about keys. I can start on changing it right now in this branch, or merge this and create a new ticket?

Reply:
> This happens in more (actually a lot of) places, also in the chart js code, that's why I was a bit reluctant to change all ids into keys. It might take me a few hours to sort it all out. 

I don't think you should fix every occurence in the codebase at this time but this PR will add couple of lines diff of the future fix. Fixing this controller and its routes shouldn't take an hour and will make sure you wont have to check over this file in the future.

Reply:
This is now changed